Childrens-Social-Care-CPD/Childrens-Social-Care-CPD.csproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Properties\" />
+    <Folder Include="wwwroot\assets\images\moj\" />
```

Comment:
Will these be removed when we have the final branding assets?

Reply:
Possibly. Out use of the filter component from the MOJ package isn't final so they might get removed. Otherwise they're all there in support of the MOJ CSS.